### ISSUE TITLE
UIA instability when reconnecting on bad network

### ISSUE THREAD

**Issue by Danstiv** [BearWare/TeamTalk5#3211]
### Description

NVDA unable to interact with uia apps during reconnection with broken network, it is often occurs when working with the network configuration of a router via ssh, the user has a uia application (windows terminal), tt is running in the background, and there is also a need to frequently reboot the router or network configuration to check the changes made.
At some point, DNS breaks down, or all traffic stops passing, or windows refuses to reconnect to wi-fi, and in such cases tt tries to reconnect to the server.
And while the user is trying to solve the problem via ssh, tt does something with uia, and nvda stops reading content from the windows terminal.
Also, for example, nvda cannot interact with unigram (uia app) during such lag, i.e. this problem is not specific to tt and windows terminal.
Another symptom of the problem: tt responds with a delay to the ptt key, for several seconds nothing happens after several presses, then a lot of ptt sounds are played, then again no response to the key.
In other words, the main thread of the program is probably hanging.

### Application

- [x] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [ ] TeamTalkServer

### Platform

- [x] Windows
- [ ] macOS
- [ ] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

Apps should respond normally.

### Actual behavior

NVDA unable to interact with uia application.

### Steps to reproduce problem

1. Connect to the server.
2. Turn your network in bad state, for example connect to a wireguard tunnel with invalid parameters so connection never established.
3. Wait for disconnection from tt server.
4. During autoreconnect try to interact with uia apps using nvda screenreader.

**Comment by BlindMaster24** [comment]
Yeah!

**Comment by bear101** [comment]
This is due to the inherit blocking of [`getaddrinfo`](https://github.com/BearWare/TeamTalk5/blob/master/Library/TeamTalkLib/myace/MyINet.cpp#L59C39-L59C50). We don't plan to change this.
